.github/workflows/release.yml

```
@@ -45,6 +45,24 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
+      - name: Check Version Consistency
+        id: version_check
+        run: |
+          BRANCH_VERSION="${{ env.VERSION }}"
+          
+          # Safely extract __version__ from the file (for example, '1.0.0' from __version__ = '1.0.0')
+          CODE_VERSION=$(grep '__version__' src/docbuild/__about__.py | awk -F "'" '{print $2}')
+          
+          echo "Branch version: $BRANCH_VERSION"
+          echo "Code version:   $CODE_VERSION"
```

I would use a "notice" command from the [Workflow commands](https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-commands#setting-a-notice-message) page:


Syntax:
```
::notice file={name},line={line},endLine={endLine},title={title}::{message}
```

```suggestion
          echo ":notice title=Branch version::$BRANCH_VERSION"
          echo ":notice title=Code version::$CODE_VERSION"
```